### PR TITLE
Move Perl commands to separate directory

### PR DESCRIPTION
**Pull request by richsalz**
We do not want to require Perl to be present on all systems where QuicTLS is installed.  And these two commands are not used during build/test.

Also, I think I have ported the `rehash` command to windows, so once this is approved, I can move c_rehash.pl here also.

## Checklist
Thank you for your contribution. Please answer the following:

- [X] I acknowledge that I am authorized to submit this code under
the terms of the [Apache License](https://www.apache.org/licenses/LICENSE-2.0)
- [ ] All new public APIs are documented
- [ ] All new public functions have tests
- [ ] All new public functions have fuzzing tests

### CODE REVIEW

**Review by wbl — APPROVED** [review]

**Comment by richsalz** [comment]
I want to wait until I get #136 working and then I'll merge this because otherwise the windows build is broken.

**Comment by richsalz** [comment]
Not going to wait for #136, as this removes two more `.in` (perl template) files.

**Review by wbl — APPROVED** [review]

**Pull request merged**
